test/cases/light-dom/src/pages/index.js

```diff
@@ -7,10 +7,15 @@ export default class HomePage extends HTMLElement {
   }
 
   getTemplate() {
-    return `
+    return `<!DOCTYPE html>
```

Comment:
since we have dedicated test cases for the wrapping tags, maybe we can leave this test case as is?